tests/README.md

```diff
@@ -0,0 +1,22 @@
+# IBM Analog Hardware Acceleration Kit: Unit Tests
+
+
+This project contains a unittest compatible test suite, that can be executed by any Python test runner. The recommended runner is pytest, which can be installed along with a number of other development tools used in aihwkit via:
+```
+pip install -r requirements-dev.txt 
+```
+
+All the tests and example do not have a main.py file. To run the full test suite simply do (from the command line):
```

Comment:
Maybe just remove the sentence "All the tests and example do not have a main.py file."